experimental/dds/tree/src/test/EditUtilities.tests.ts

```diff
@@ -195,7 +195,7 @@ describe('EditUtilities', () => {
 				traits: { [leafTrait]: [testTree.buildLeaf(leafId)] },
 			};
 			const clone = convertTreeNodes<ChangeNode, ChangeNode>(tree, (node) => ({ ...node }));
-			assert(typeof clone !== 'number', '');
+			assert(typeof clone !== 'number', 0x660 /*  */);
```

Comment:
looks like this test file is using the wrong assert. maybe worth fixing here?

Reply:
if it used some other assert library, none of these would even compile any more (the assert tagging ahs no idea what assert library you are using, it just looks for "assert"). Most tests escape harm by not having messages on their asserts.

Reply:
Really what wrong with this line is explicitly having an empty string assert message, which is not very useful.

Could remove the messages from this and the others (then use a different assert library), or convert them into expect. I'm not sure there would be a benefit to doing that.